src/FactSystem/FactMetaData.h

```diff
@@ -148,6 +148,9 @@ class FactMetaData : public QObject
     /// Used to add new values to the enum lists after the meta data has been loaded
     void addEnumInfo(const QString& name, const QVariant& value);
 
+    /// Used to remove values from the enum lists after the meta data has been loaded
+    void rmEnumInfo(const QString& name);
```

Comment:
Can you name is `removeEnumInfo`? This needs to work from the value not the string. The strings are not unique since they can be translated. Only the values are unique.

Reply:
Thanks for pointing this out, the last version contains the requested changes.